apps/condo/domains/subscription/schema/ServiceSubscription.test.js

```diff
@@ -507,20 +507,10 @@ describe('ServiceSubscription', () => {
             const adminClient = await makeLoggedInAdminClient()
             const [organization] = await createTestOrganization(adminClient)
 
-            const [obj, attrs] = await createTestServiceSubscription(adminClient, organization)
+            await createTestServiceSubscription(adminClient, organization)
 
             const objs = await ServiceSubscription.getAll(adminClient, {}, { sortBy: ['updatedAt_DESC'] })
             expect(objs.length >= 1).toBeTruthy()
```

Comment:
There are a number of calls of `createTestServiceSubscription` during tests. That's why the `objs.lenght` always will be bigger than 1.
<img width="1282" alt="image" src="https://user-images.githubusercontent.com/25384290/162025441-0554c58b-b0c9-42b6-87b6-c6fd29ebf769.png">
Looks like this test never fails.

What about checking at least `organization.id`?

Reply:
It is not an objective of this test case. We are checking an access only, not the role-based scopes.